iam.tf

```diff
@@ -62,7 +62,7 @@ data "aws_iam_policy_document" "ssm_parameters_access" {
     ]
 
     resources = [
-     "arn:aws:ssm:*:*:parameter${var.parameter_store_path}*"
+      "arn:aws:ssm:*:*:parameter${var.parameter_store_path}*"
```

Comment:
There appears to be some rouge formatting included in some files. Can you please run a `terraform fmt -recursive` at the root of the repo and add a commit with these format changes only. That will tidy things up and may eliminate redundant changes

Reply:
> There appears to be some rouge formatting included in some files. Can you please run a `terraform fmt -recursive` at the root of the repo and add a commit with these format changes only. That will tidy things up and may eliminate redundant changes

Ran the command. No change in format:)